.ci/download_db.sh

```diff
@@ -0,0 +1,2 @@
+cd "${0%/*}"
+wget 'https://github.com/virresh/byld-website/raw/db/planet.db' -O ../site/planet.db || true
```

Comment:
Why is this needed? From what I understand, `planet.db` stores the local entries. 

Also need clarification on the exact mechanism of blog. Feeds are generally limited to top n entries on a blog. Are we storing older entries? Do we rebuild just from the feed?

I think either is fine. Though rebuilding means "permalinks" are perma no more

Reply:
Yes, planet.db is the sqlite database that will store blog entries locally, whenever you run jekyll, it will rebuild all posts from that database

The reason why I've added this is:
Suppose you have a feed that you added today, and removed in another PR, if we do not store the database as an output product somewhere, we will end up losing those blogs, since we will only look at top n feeds that are currently in the .ini


Also permalinks don't make sense for blog entries written by others, since they have the original content already hosted at someplace, we are just providing a means to read them on our site giving them possibly a better visibility

Reply:
Also when you run the update command, that is the time when all these feeds are revisited and the database updated

Reply:
Cool. Please mention that this script takes a backup of the db before rebuilding it to preserve the older entries which may no longer be in feeds as a comment somewhere.